# Bulgarian translation of update manager.
# Copyright (C) 2005 THE update manager'S COPYRIGHT HOLDER
# This file is distributed under the same license as the update manager package.
# Rostislav "zbrox" Raykov <zbrox@i-space.org>, 2005.
#
#
msgid ""
msgstr ""
"Project-Id-Version: update manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-08-16 22:34+0000\n"
"Last-Translator: Gangov <Unknown>\n"
"Language-Team: Bulgarian <dict@fsa-bg.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:52+0000\n"
"X-Generator: Launchpad (build 14157)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr "0 кБ"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr "1 кБ"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr "%.0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f МБ"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "Сървър за %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Основен сървър"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Сървъри по избор"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "Не може да пресметне запис в sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Намирането на програмни пакети не бе успешно. Вероятно това не е диск с "
"Убунту или е с неподходяща архитектура за вашата система?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "Грешка при добавяне на CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Възникна грека при добавянето на CD, надграждането ще бъде прекратено. Моля "
"докладвайте това като грешка ако става въпрос за оригинален Убунту компакт "
"диск.\n"
"\n"
"Съобщението за грешка беше:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Премахване на пакет в лошо състояние"
msgstr[1] "Премахване на пакети в лошо състояние"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Пакетът '%s' е в нестабилно състояние и трябва да се преинсталира, но не "
"може да се намери архив за него. Искате ли да премахнете този пакет сега, за "
"да продължите?"
msgstr[1] ""
"Пакетите '%s' са в нестабилно състояние и трябва да се преинсталират, но не "
"могат да се намерят архиви за тях. Искате ли да ги премахнете сега, за да "
"продължите?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Сървърът може би е претоварен"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Повредени пакети"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Системата Ви съдържа повредени пакети, които не могат да бъдат поправени с "
"този софтуер. Моля, поправете ги със synaptic или apt-get преди да "
"продължите."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"При изчисляване на надграждането възникна нерешим проблем:\n"
"%s\n"
"\n"
" Това може да се случи от:\n"
" * Надграждане до преиздадена версия на Убунту\n"
" * Изпълняване на преиздадена версия на Убунту\n"
" * Неофициални пакети, които не са предоставени от Убунту\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Това най-вероятно е временен проблем. Моля, опитайте отново по-късно."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Ако нищо от това не се приложи, тогава моля докладвайте тази грешка "
"използвайки командата 'ubuntu-bug update-manager' във терминал."

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Не може да бъде изчислено надграждането"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Грешка при удостоверяването автентичността на някои пакети"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Не беше възможно да се удостовери автентичността на някои пакети. Това може "
"да е временен проблем с мрежата. Може би бихте искали да опитате отново по-"
"късно. Вижте по-долу списъка на неудоствоерените пакети."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Пакетът '%s' е маркиран за премахване, но е в черния списък за премахване."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Важният пакет '%s' е маркиран за премахване."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Опит за инсталиране на забранена версия на '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "Не може да се инсталира '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Беше невъзможно да се инсталира исканият пакет. Моля докладвайте тази грешка "
"използвайки 'ubuntu-bug update-manager' във терминал."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Не могат да бъдат предположени мета-пакети"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Системата Ви няма инсталиран ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop или edubuntu-desktop пакет и не бе възможно засичането на версията "
"на Убунту, която ползвате.\n"
" Моля, преди да продължите, първо инсталирайте един от тези пакети, като "
"използвате synaptic или apt-get!"

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Четене от кеша"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Неуспех при получаване на изключителни права"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Това обикновено означава, че е стартирана друга система за управление на "
"пакети (като apt-get или aptitude). Първо спрете тази програма."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Надграждането през отдалечена връзка не се поддържа"

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Вие пускатe надграждане през отдалечена ssh връзка с фронтенд, който не "
"поддържа това. Моля опитайте надграждане в текстов режим с  'do-release-"
"upgrade'.\n"
"\n"
"Надграждането ще бъде спряно сега. Моля опитайте отново без ssh."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "Продължаване на работата със SSH?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Тази сесия изглежда работи под ssh. Не е препоръчително да се изпълнява "
"надграждане през ssh в момента, защото в случай на неуспех възстановяването "
"е по-трудно. \n"
"\n"
"Ако продължите, допълнителен ssh демон ще бъде стартиран на порт '%s'.\n"
"Искате ли да продължите?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Стартиране на допълнителен SSHD"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"За да направите възстановяването по-лесно в случай на загуба на данни, ще "
"бъде стартиран допълнителен sshd на порт '%s'. Ако нещо се обърка с "
"изпълняващия се ssh, ще можете да се свържете към допълнителния ssh.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Ако използвате защитна стена, може да се наложи временно да отворите този "
"порт. Тъй като това е потенциално опасно, не е направено автоматично. Може "
"да отворите порта с например:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Надграждането невъзможно"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Надграждане от '%s' на '%s' не се поддържа с този инструмент."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "Настройката на ограничителен режим е неуспешна"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Не беше възможно да се създаде среда за ограничителен режим."

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "Ограничителен режим"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Това надграждане се изпълнява в ограничителен (пробен) режим. Всички промени "
"се записват в '%s' и всички промени ще бъдат изгубени при следващото "
"рестартиране.\n"
"\n"
"*Не* промените, записани в systemdir от сега до следващото рестартиране."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Инсталацията на Python е повредена. Моля, поправете симболната връзка "
"„/usr/bin/python”!"

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "Пакетът 'debsig-verify' е инсталиран"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Надграждането не може да продължи с инсталиран този пакет.\n"
"Моля, премахнете го със synaptic или 'apt-get remove debsig-verify' и после "
"отново изпълнете надграждането."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Включване на най-новите актуализации от Интернет?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Системата за надграждане може автоматично да използва Интернет, за да "
"изтегли най-новите актуализации и да ги инсталира по време на надграждането. "
"Ако имате връзка с Интернет, това се препоръчва.\n"
"\n"
"Това надграждане ще отнеме повече време, но когато завърши, вашата система "
"ще бъде с инсталирани най-новите актуализации. Може да не искате да "
"направите това, но трябва да инсталирате най-новите актуализации след като "
"надграждането завърши.\n"
"\n"
"Ако отговора ви е 'Не', мрежата няма да се използва изобщо."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "забранен при надграждане до %s"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Не е открит валиден огледален сървър"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"При сканирането на хранилището, не беше намерено огледало за надграждане. "
"Това може да се случи, когато използвате вътрешно огледало или информацията "
"за огледалото е изтекла.\n"
"\n"
"Искате ли въпреки това да презапишете файла 'sources.list'? Ако натиснете "
"'Да' ще се актуализират всички '%s' до '%s' записи.\n"
"Ако натиснете 'Не', няма да се извърши надграждане."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Генериране на източници по подразбиране?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"След сканирането на 'sources.list', не беше намерен валиден запис за '%s'.\n"
"\n"
"Да се добавят ли записите по подразбиране за '%s'? Ако натиснете 'Не', няма "
"да се извърши надграждане."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Информацията от хранилището е невалидна"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""
"Ъпгрейдвайки информацията в хранилището доведе до невалиден файл. Моля "
"докладвайте тази грешка използвайки командата  'ubuntu-bug update-manager' в "
"терминал."

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Забранени са източници от трети страни"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Някои записи от трети страни във Вашия „sources.list” бяха забранени. Можете "
"да ги разрешите отново след надграждането чрез инструмента „software-"
"properties” или чрез диспечера Ви на пакети."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакет в несъвместимо състояние"
msgstr[1] "Пакети в несъвместимо състояние"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Пакетът '%s' е в несъвместимо състояние и трябва да се преинсталира, но не "
"може да се намери архив. Моля, преинсталирайте пакета ръчно или го "
"премахнете от системата."
msgstr[1] ""
"Пакетите '%s' са в несъвместимо състояние и трябва да се преинсталират, но "
"не могат да се намерят архиви. Моля, преинсталирайте пакетите ръчно или ги "
"премахнете от системата."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Грешка по време на надграждане"

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Възникна проблем при актуализиране. Това обикновено се дължи на мрежов "
"проблем. Моля, проверете вашата мрежова връзка и опитайте отново."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Недостатъчно свободно място на диска"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Надграждането бе прекъснато. Необходими са %s свободно пространство на диск "
"'%s'. Моля, освободете поне %s допълнително пространство на диск '%s'. "
"Изчистете си кошчето и премахнете временни пакети от предишни инсталации "
"използвайки 'sudo apt-get clean'."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Пресмятане на промените"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Желаете ли да започне надграждането?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Надграждането е отменено"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Сега надграждането ще бъде отказано и ще се възстанови оригиналното "
"състояние на системата. Можете да продължите надграждането по-късно."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Не могат да бъдат свалени обновленията."

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Актуализацията беше прекъсната. Моля проверете интернет връзката си или "
"медията, от която инсталирате и опитайте отново. Всички свалени файлове са "
"запазени."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Грешка при прехвърляне"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Възстановяване на първоначалното състояние на системата"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Не можаха да бъдат инсталирани обновленията"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Надграждането бе прекъснато. Системата ви може да е в нестабилно състояние. "
"Сега ще се извърши възстановяване (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Моля докладвайте тази грешка използвайки командата 'ubuntu-bug update-"
"manager' във терминал и включете файловете, който се намират в /var/log/dist-"
"upgrade/ в доклада Ви за грешка.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Надграждането бе прекъснато. Моля, проверете Интернет връзката или "
"инсталационната среда и опитайте отново. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Премахване на остарелите пакети?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Задържане"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Премахване"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Възникна проблем при почистване. Моля, вижте съобщението отдолу за повече "
"информация. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Не е инсталирана изисквана зависимост"

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Изискваната зависимост „%s” не е инсталирана. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Проверка на диспечера на пакети"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Подготовката за надграждането неуспешна"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Подготвянето на системата за ъпгрейда се провали. Моля докладвайте тази "
"грешка използвайки командата 'ubuntu-bug update-manager' в терминал и "
"включете файловете, които се намират в /var/log/dist-upgrade/ в доклада Ви "
"за грешка."

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Грешка при получаване на изискванията за надграждане"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Системата не можа да получи предпоставките за ъпгрейда. Ъпгрейдът ще "
"прекъсне сега и ще възстанови оригиналното системно състояние."

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Актуализиране информацията от хранилището"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Неуспех при добавянето на cdrom"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Съжаляваме, добавянето на cdrom не беше успешно."

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Невалидна информация за пакета"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"След като вашата пакетна информация беше актуализирана пакетът '%s' не може "
"да бъде намерен вече.\n"
"Това показва сериозна грешка, моля докладвайте тази грешка използвайки "
"командата 'ubuntu-bug update-manager' в терминал и включете файловете, които "
"се намират в /var/log/dist-upgrade/ в доклада Ви за грешка."

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Получаване"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Надграждане"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Надграждането е завършено"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Надграждането беше завършено, но възникнаха грешки по време на този процес."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Търсене на остарял софтуер"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Надграждането на системата завърши."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Частичното надграждане завърши."

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "evms се използва"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Вашата система използва мениджър на устройства 'evms' в /proc/mounts. "
"Софтуерът 'evms' вече не се поддържа. Моля изключете го и изпълнете "
"надграждането отново."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr ""
"Вашият графичен хардуер може да не бъде напълно поддържан в Убунту 11.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""
"Поддръжката в Убунту 11.04 за вашият Интел графичен хардуер е ограничена и "
"може да се сблъскате с проблеми след актуализацията. Искате ли да продължите "
"с актуализацията?"

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Надграждането може да намали ефектите на работния плот и производителността "
"в игрите и други графични програми."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Този компютър в момента използва графичния драйвер NVIDIA 'nvidia''. Няма "
"версия на този драйвер, която да работи с вашата видео карта под Убунту "
"10.04 LTS.\n"
"\n"
"Искате ли да продължите?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Този компютър използва графичния драйвер AMD 'fglrx'. Няма версия на този "
"драйвер, която да работи с вашия хардуер под Убунту 10.04 LTS.\n"
"\n"
"Искате ли да продължите?"

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "Не е i686 архитектура"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Вашата система използва i586 процесор или такъв който няма разширение "
"'cmov'. Всички пакети са създадени с оптимизации, изискващи i686 като "
"минимална архитектура. Не е възможно да ъпгрейднете своята система към ново "
"издание на Убунту с този хардуер."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "Няма ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Вашата система ползва ARM CPU, която е по-стара от ARMv6 архитектура. Всички "
"пакети в Karmic бяха оптимизрани с изискване минимум ARMv6 архитектура. Не е "
"възможно да надграждате вашата система с новата версия на Убунту с този "
"хардуер."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "Няма достъпен init"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Вашата система изглежда, че е виртуализирана среда без init процес, т.е. "
"Linux-VServer. Убунту 10.04 LTS не може да функционира в този тип среда, "
"изисква се, първо, обновяване на настройката на Вашата виртуална машина.\n"
"Сигурни ли сте, че искате да продължите?"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "Sandbox надграждане с помощта на aufs"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Ползване на зададения път за търсене на CD дискове с надграждаеми пакети"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Ползване на фронтенд. Текущо налични:\n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr "*НЕПРЕПОРЪЧИТЕЛНО* тази опция ще бъде игнорирана"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Извършване само на частично надграждане (без презаписване на sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "Изключване поддръжката на GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Задаване на datadir"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Моля, поставете диск \"%s\" в устройство \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Изтеглянето е завършено"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Изтегляне на файл %li от %li с %sB/с"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Остават около %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Изтегляне на файл %li от %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Прилагане на промените"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "проблеми със зависимости - остава не конфигуриран"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Не можеше да бъде инсталиран \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Надграждането ще продължи, но '%s\" пакетът може да не работи. Моля "
"помислете за представянето на грешки"

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Замяна на персонализиран конфигурационен файл\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ще изгубите всички промени, които сте направили в този конфигурационен файл "
"ако изберете да го замените с по-нова версия."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "Командата \"diff\" не бе намерена"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Възникна фатална грешка"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Моля, докладвайте това като бъг (ако вече не сте) и включете файловете "
"/var/log/dist-upgrade/main.log и /var/log/dist-upgrade/apt.log в доклада. "
"Надграждането бе прекъснато.\n"
"Оригиналният файл sources.list бе запазен в "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Ctrl-c натиснат"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Това ще прекрати операцията и може да остави системата в повредено "
"състояние. Сигурни ли сте, че искате да направите това?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"За да предодвратите загуба на данни, затворете всички отворени приложения и "
"документи."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Вече не се поддържа от Каноникал. (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Подграждане (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Премахнете (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Вече не е необходим (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Инсталирайте (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Надграждане (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Смяна на носител"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Показване на разлики >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Скриване на разлики"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Грешка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "&Отказ"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Затваряне"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Показване на терминал >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Скриване на терминал"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Информация"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Детайли"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Вече не се поддържа %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "Премахване на %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Премахване (на автоматично инсталиран) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "Инсталиране на %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "Надграждане на %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Нужно е рестартиране"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Рестратирайте системата, за да завършите надграждането</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Рестартирай сега"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Отказване на изпълняващото се надграждане?</big></b>\n"
"\n"
"Ако откажете надграждането системата може да остане в състояние, в което е "
"неизползваема. Силно ви препоръчваме да подължите надграждането."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Отмяна на надграждането?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li ден"
msgstr[1] "%li дни"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li час"
msgstr[1] "%li часа"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li минута"
msgstr[1] "%li минути"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li секунда"
msgstr[1] "%li секунди"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Това изтегляне ще отнеме около %s с 1Mbit DSL връзка и около %s с 56k модем."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Това изтегляне ще отнеме около %s с вашата връзка. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Подготовка за надграждане"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Получаване на нови софтуерни канали"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Получаване на нови пакети"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Инсталиране на надгражданията"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Почистване"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d инсталиран пакет вече не се поддържа от Каноникал. Все още можете "
"да получите поддръжка от обществеността."
msgstr[1] ""
"%(amount)d инсталирани пакета вече не се поддържат от Каноникал. Все още "
"можете да получите поддръжка от обществеността"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d пакет ще бъде премахнат."
msgstr[1] "%d пакети ще бъдат премахнати."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d нов пакет ще бъде инсталиран."
msgstr[1] "%d нови пакети ще бъдат инталирани."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d пакет ще бъде надграден."
msgstr[1] "%d нови пакети ще бъдат надградени."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Ще трябва да бъдат изтеглени общо %s. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Изтеглянето и инсталирането на надграждане може да отнеме няколко часа. Щом "
"инсталирането завърши, процесът не може да бъде отказан."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "Системата Ви е актуална"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Няма налични надграждания за вашата система. Надграждането ще бъде "
"прекратено."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Нужно е рестартиране"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "Надграждането е завършено и има нужда от рестарт. Рестартиране сега?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Не може да бъде стартирана помощната програма за надграждане."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Това е най-вероятно грешка в инструмента за ъпгрейд. Моля докладвайте тази "
"грешка използвайки командата 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Подпис на помощната програма за надграждане"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Помощна програма за надграждане"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Грешка при доставянето"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Доставянето на надграждането бе неуспешно. Възможно е да има проблем с "
"мрежата. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Грешка при извличането"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Извличането на надграждането бе неуспешно. Възможно е да има проблем с "
"мрежата или сървъра. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Проверката е неуспешна"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Проверката на надграждането се провали. Възможно е да има проблем с мрежата "
"или със сървъра. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Идентификацията неуспешна"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Идентификацията на надграждането е неуспешна. Възможно е да има проблем с "
"мрежата или сървъра. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Не може да се изпълни надграждане"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Това обикновенно е причинено от система, където /tmp е монтиран noexec. Моля "
"ремонтирайте без noexec и стартирайте ъпгрейда отново."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Съобщението с грешка е '%s'."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Моля, докладвайте това като бъг и включете файловете /var/log/dist-"
"upgrade/main.log и /var/log/dist-upgrade/apt.lo в доклада. Надграждането бе "
"прекъснато.\n"
"Оригиналният файл sources.list бе запазен в "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Прекратяване"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Понижен:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "За да продължите моля натиснете [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Продъжаване [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Детайли [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "Y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Вече не се поддържа: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Премахване на %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Инсталиране: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Надграждане на %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Продължаване [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"За приключване на актуализирането е необходимо рестартиране.\n"
"Ако изберете 'y' системата ще се рестартира."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Отмяна на протичащото надграждане?</big></b>\n"
"\n"
"Системата Ви може да се окаже в неузползваемо състояние, ако отмените "
"надграждането. Силно препоръчвано е да възобновите надграждането!"

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Рестартирайте системата за да завършите надграждането</big></b>\n"
"\n"
"Моля запишете текущата си работа преди да продължите."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Започване на надграждането?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr "<b><big>Актуализиране на Убунту до версия 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Разлика между файловете"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Надграждане на дистрибуцията"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Рестартиране на компютъра"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Настройване на нови софтуерни канали"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Терминал"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Отмяна на надграждането"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Продъжаване"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Замяна"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Съобщи грешка"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Поднови надграждането"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Стартиране на надграждането"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Моля, изчакайте! Това може да отнеме известно време."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Актуализацията е завършена"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Не бяха открити бележки към изданието"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Сървърът може да е претоварен. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Не можеше да бъдат свалени бележките към изданието"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Моля, проверете Интернет връзката си!"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Надграждане"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Бележки към изданието"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Изтегляне на допълнителни пакетни файлове..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s от %s с %sB/с"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "Файл %s от %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Отваряне на връзката в браузър"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Копиране на връзката в клипборда"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Смъкване на файл %(current)li от %(total)li при %(speed)s/сек"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Изтегляне на файл %(current)li от %(total)li"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Вашата версия на Убунту вече не се поддържа."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Повече няма да получавате никакви поправки по сигурността или критични "
"обновления. Моля надградете до по-нова версия на Убунту Линукс."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Информация за надграждане"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "Версия %s: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Не е засечена мрежова връзка. Не можете да изтеглите файла с промените."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Изтегляне на списъка с промени..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr "_Премахване на избора от всички"

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr "Избери_Всичко"

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "%(count)s актуализация е избрано."
msgstr[1] "%(count)s актуализации са избрани."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s ще бъде изтеглено."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "Актуализацията вече е свалена, но не е инсталирана."
msgstr[1] "Актуализациите вече са свалени, но не са инсталирани."

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Няма актуализации за инсталиране."

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Неизвестен размер за изтегляне"

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Не е известно кога последно е актуализирана пакетната информация. Моля, "
"натиснете \"Провери\" бутона, за да актуализирате информацията."

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Информацията за пакетите беше обновена преди %(days_ago)s дни.\n"
"Натиснете бутона 'Проверка', за да проверите за софтуерни обновления."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] "Пакетът информация бе последно обновен преди %(days_ago)s ден."
msgstr[1] "Пакетът информация бе последно обновен преди %(days_ago)s дена."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Информацията за пакета последно е обновена преди %(hours_ago)s час."
msgstr[1] ""
"Информацията за пакета последно е обновена преди %(hours_ago)s часа."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Пакетната информация беше актуализирана преди около %s минути."

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr "Пакетната информация беше току що актуализирана."

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Софтуерните актуализации поправят грешки, премахват уязвими места в "
"сигурността и предлагат нови възможности."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr "Може да има актуализации за вашия компютър."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Добре дошли в Убунту"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""
"Тези софтуерни актуализации са издадени, откакто е пусната тази версия на "
"Убунту. Ако не искате да ги инсталирате сега, изберете \"Мениджър на "
"актуализациите\" от Апликации по-късно,"

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""
"Тези софтуерни актуализации са издадени, откакто е пусната тази версия на "
"Ubuntu. Ако не искате да ги инсталирате сега, изберете \"Мениджър на "
"актуализациите\" от Администраторското меню по-късно."

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr "Достъпни са актуализации за този компютър"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Ако не желаете да ги инсталирате сега, по-късно изберете \"Диспечер на "
"обновленията\"."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Ако не искате да ги инсталирате сега, по-късно изберете 'Диспечер на "
"актуализациите' от меню Администриране."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Надграждането се нуждае от общо %s свободно място на диск '%s'. Моля, "
"освободете поне %s място на диск '%s'. Изпразнете вашето кошче и премахнете "
"временните пакети от инсталации с командата 'sudo apt-get clean'."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"За да завърши инсталирането, компютъра трябва да се рестартира. Преди да "
"продължите, моля, запазете вашата работа."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Прочитане на информация за пакет"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Свързване..."

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Възможно е да не можете да проверите за актуализации или изтегляне на нови "
"такива."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Невъзможно инициирането на информацията за пакета"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"При инициализиране на информация за пакетите възникна нерешим проблем.\n"
"\n"
"Моля, докладвайте това като бъг в пакета 'update-manager' и включете "
"следното съобщение с грешка:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"При изчисляване на надграждането възникна нерешим проблем.\n"
"\n"
"Моля, докладвайте това като бъг в пакета 'update-manager' и включете "
"следното съобщение с грешка:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (Нова инсталация)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Размер: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "От версия %(old_version)s към %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Версия %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Надграждане към ново издание не е възможно в момента"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Надграждането към ново издание понастоящем не може да се извърши, моля "
"опитайте по-късно. Сървърът върна: \"%s\""

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Сваляне на инструмента за награждане на версията"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Налично е ново издание на Убунту - '%s'</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Индексът на софтуера е повреден"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Невъзможно е да бъде инсталиран или премахнат какъвто и да е софтуер. Моля, "
"ползвайте диспечера на пакети \"Synaptic\" или първо задействайте \"sudo apt-"
"get install -f\" в терминален прозорец, за да поправите този проблем!"

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Провери за актуализации"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Инсталирай всички възможни актуализации"

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Отказ"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Инсталиране"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Изграждане на списък с актуализации"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Нормално надграждане не може да бъде изчислено, моля изпълнете: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Това може да се случи поради:\n"
" * Предишно надграждане, което не е завършило\n"
" * Проблеми с част от инсталирания софтуер\n"
" * Неофициални пакети, които не са предоставени от Убунту\n"
" * Нормални промени в предварителна версия на Убунту"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Изтегляне на отчет с промените"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Други актуализации (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Промени за версиите:\n"
"Инсталирана версия: %s\n"
"Възможна версия: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr "Тази актуализация не идва от източник, който поддържа промени."

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Неуспех при изтегляне на списъка с промени. \n"
"Моля, проверете Интернет връзката си."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Отчета с промени не съдържа никакви промени.\n"
"\n"
"Моля, използвайте http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"докато промените станат достъпни или опитайте отново по-късно."

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Списъкът с промени все още не е достъпен.\n"
"\n"
"Моля, използвайте http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"докато промените станат достъпни или опитайте отново по-късно."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Неуспешен опит за откриване на дистрибуцията"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Възникна грешка '%s' докато се проверяваше каква система използвате."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Важни актуализации за сигурността"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Препоръчителни актуализации"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Предложени актуализации"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Състарени версии"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Актуализации на дистрибуцията"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Други актуализации"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Трябва ръчно да проверите за актуализации</big></b>\n"
"\n"
"Вашата система не проверява автоматично за актуализации. Можете да промените "
"това в <i>Източници на софтуер</i> в раздела <i>Актуализации</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr ""
"<big><b>Не всички актуализации могат да бъдат инсталирани</b></big>\r\n"
"\r\n"
"%s"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Работа на батерия</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<b><big>Стартиране на Диспечера на актуализации</big></b>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Промени"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "Провер_ка"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "Продъл_жаване"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Описание"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Описание на актуализацията"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Преди да започне актуализирането по-добре да свържете захранващия адаптер."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Частично надграждане, за да се инсталират възможно най-много актуализации. \n"
"\n"
"Това може да се случи заради:\n"
" * Предишно надграждане, което не е завършило\n"
" * Проблеми с някой от инсталирания софтуер\n"
" * Неофициални пакети, които не са предоставени от Убунту\n"
" * Нормални промени от предварителна версия на Убунту"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Показване на напредък за индивидулни файлове"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Стартиране на диспечера за актуализациите"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "_Надграждане"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Диспечер на актуализациите"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Вие сте свързани чрез безжичен модем"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Вие сте свързани през роуминг и може да бъдете таксувани за данните, "
"изразходвани за това обновление."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Вашата система работи на батерия. Сигурни ли сте, че искате да продължите?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "_Скрий тази информация за в бъдеще"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "_Инсталиране на актуализациите"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "_Частично надграждане"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr "_Настройки..."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Надграждане"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "актуализации"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Налична е нова версия на Убунту. Бихте ли искали да надградите?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Напомняне по късно"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Не надграждай"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Да, надгради сега"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Можете да надградите по-късно като отворите Диспечера за Надграждане и "
"щракнете върху \"Надгради\"."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Вие сте отхвърлили надграждането до нова версия на Убунту"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Показване и инсталация на наличните актуализации"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Обновления на програмите"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Показване на версията и изход"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Папка, която съдържа файловете с данни"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Проверка дали има по-ново издание на Убунту"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Проверка дали е възможно надграждане до последното издание в разработка"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Надграждане с предложената най-нова весрия"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Да не се фокусира на карта при стартиране"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "Опит за пускане на dist-upgrade"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Да не се проверява за обновления при пускане"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Тестване на надграждане в ограничителен режим aufs припокриване"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Протичане на частично надграждане"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Опит за надграждане до най-новата версия с помощта на програма за "
"надграждане от $distro-proposed"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Изпълнение в специален режим за надграждане.\n"
"Текущо се поддържат 'работен плот' за редовно актуализиране на настолни "
"системи и 'сървър' за сървърни системи."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Стартиране на определения фронтенд"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Проверка само ако нова дистрибуция е налична и доклаване на резултата чрез "
"exit код"

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Проверка за нова версия на Убунту"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"За повече информация, моля посетете:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Не е намерена нова дистрибуция"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "Достъпно е ново издание на '%s'."

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Изпълнете 'do-release-upgrade', за да го надградите."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Убунту %(version)s налична за надграждане"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Вие сте отказали надграждането до Убунту %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Неимплементиран метод: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Файл на диск"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb пакет"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Инсталиране на липсващ пакет."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Пакетът %s не трябва да се инсталира."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i остарели записа във файла за състояние"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Остарели записи в състоянието на dpkg"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Остарели записи в състоянието на dpkg"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Когато се надгражда, ако kdelibs4-dev е инсталиран, ще трябва да се "
"инсталира и kdelibs5-dev. Вижте bugs.launchpad.net, bug #279621 за повече "
"информация."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s трябва да се маркира като ръчно инсталиран."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Премахване на lilo докато и grub е инсталиран.(Вижте грешка #314004 за "
"подробности.)"

#~ msgid "0 KB"
#~ msgstr "0 КБ"

#~ msgid "The window size"
#~ msgstr "Размерът на прозореца"

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr ""
#~ "Съхранява състоянието на разширения панел, който съдържа списъка с промени и "
#~ "описанието"

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "Съхранява размера на update-manager диалога"

#~ msgid "Show details of an update"
#~ msgstr "Показване подробностите за актуализациите"

#~ msgid "Remind to reload the channel list"
#~ msgstr "Припомняне за презареждане на списъка с канали"

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Ако е забранена автоматичната проверка за актуализации, ще трябва ръчно да "
#~ "презареждате списъка с канали. Тази опция позволява в този случай да се "
#~ "скрие напомнянето."

#~ msgid "_Check All"
#~ msgstr "_Проверка на всички"

#~ msgid "_Uncheck All"
#~ msgstr "_Отморкирай всичко"

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr ""
#~ "Това най-верроятно е грешка в помощната програма за надграждане. Моля, "
#~ "съобщете това като грешка!"

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "Надграждане на информацията от хранилището доведе до невалиден файл. Моля, "
#~ "съобщете това като грешка!"

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f КБ"

#~ msgid "1 KB"
#~ msgstr "1 КБ"

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*НЕОДОБРЕН* тази опция ще бъде игнорирана"

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "Неизвестно е, когато за последно е била обновена информацията за пакет. "
#~ "Моля, опитайте да натиснете бутона \"Проверка\", за да се обнови "
#~ "информацията."

#~ msgid "Previous selected"
#~ msgstr "Предишно избран"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "Автоматично затваряне на прозореца след успешно инсталиране"

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr "Информацията за пакетите беше обновена преди по-малко от час."

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr "*отхвърлен* Проверка за нови издания на дистрибуцията"

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "Този ключ е отхвърлен в полза на файла /etc/update-manager/release-upgrades. "
#~ "Автоматична проверка за нова версия на текущата дистрибуция и предложение за "
#~ "надграждане (ако е възможно)."

#~ msgid "First run welcome message"
#~ msgstr "Съобщение 'Добре дошли' при първо стартиране"

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Ако никое от тези не се прилага, моля докладвайте грешка в пакета 'update-"
#~ "manager' и включете файловете от /var/log/dist-upgrade/ в доклада."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Моля, докладвайте тази грешка в пакета 'update-manager' и включете файловете "
#~ "от /var/log/dist-upgrade/ в доклада.\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Подготовката на системата за надграждане е неуспешно. Моля, докладвайте тази "
#~ "грешка в пакета 'update-manager' и включете файловете от /var/log/dist-"
#~ "upgrade/ в доклада."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Системата не успя да вземе необходимите данни за надграждането. "
#~ "Надграждането ще бъде прекратено и ще бъде възстановено оригиналното "
#~ "състояние на системата.\n"
#~ "\n"
#~ "Моля, докладвайте тази грешка в пакета 'update-manager' и включете файловете "
#~ "от /var/log/dist-upgrade/ в доклада."

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "След като информацията за пакетите беше обновена, важният пакет '%s' вече не "
#~ "може да бъде намерен.\n"
#~ "Това сочи към сериозна грешка, моля, докладвайте тази грешка в пакета "
#~ "'update-manager' и включете файловете от /var/log/dist-upgrade/ доклада."

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr "Ако този ключ бъде зададен, ще се покаже съобщение за добре дошли."

#~ msgid "Show version in update list"
#~ msgstr "Показване на версия в списъка с актуализации"

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Промени в тази версия:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr "Тази промяна не идва от източник, който поддържа отчети с промени."

#~ msgid "Settings..."
#~ msgstr "Настройки..."

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Това обикновено се случва на системи където /tmp е монтиран noexec. Моля, "
#~ "монтирайте отново без noexec и изпълнете надграждането отново."

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Ако този ключ е настроен, прозореца за инсталация ще покаже информация за "
#~ "версията (от версия до версия)."

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Ако този ключ е настроен, прозореца за инсталация ще бъде затворен "
#~ "автоматично при успешна инсталация."

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr ""
#~ "Не можеше да бъде инсталиран нужен пакет. Моля, докладвайте това като бъг. "

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "Това перманентно ще скрие промпта за нова версия от check-new-release-gtk. "
#~ "Забележете, че малкия бутон в главния update-manager UI ще бъде все още там."

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "накарай check-new-release-gtk да игнорира определена нова версия"

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Надграждането бе прекъснато. Моля, проверете връзката си с Интернет или "
#~ "друга инсталационна среда и опитайте отново. Всички файлове свалени до сега "
#~ "се пазят."

#~ msgid "Show package summary before package name in update list"
#~ msgstr ""
#~ "Покажи обобщение на пакетите преди името на пакета в листа с актуализации"

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Ако ключът е зададен, прозорецът с основния списък ъпдейти ще покаже "
#~ "обобщение на пакетите преди името на пакета."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Надграждане на Убунту до версия 11.04</big></b>"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Тези софтуерни обновления са пуснати след издаването на Убунту. Ако не "
#~ "желаете да ги инсталирате сега, по-късно изберете \"Диспечер на "
#~ "обновленията\"."

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Тези актуализации бяха издадени, откакто Убунту беше издаден. Ако не искате "
#~ "да ги инсталирате, изберете \"Диспечер на актуализациите\" от менюто "
#~ "Администриране."
